### PR TITLE
FA-36-Update-the-UI-to-include-the-new-subscription-tiers

### PR DESCRIPTION
**Pull request by ramirezmorac2**
Merging this PR to include the new subscription tiers
Image

### CODE REVIEW

File: frontend/src/components/PaymentGateway/PaymentGateway.tsx [hunk 1]
```diff
@@ -17,18 +18,27 @@ export const SubscriptionPlans: React.FC<{ stripePromise: Promise<Stripe | null>
...
-            {
-                id: "price_1PYvHVEpF6ccgZLwn6uq6d4J",
-                name: "Enterprise Plan",
-                description: "Access to all features including premium support.",
-                price: "30.00",
-                interval: "month"
+        // Fetch product prices when the component mounts
+        async function fetchPrices() {
+            try {
+                const data = await getProductPrices({ user, product_id:"prod_R05WPWPAgXt6Kj" });
```
Comment (maacaro) [inline frontend/src/components/PaymentGateway/PaymentGateway.tsx:28]: @ramirezmorac2 this should be saved in an environment var `product_id:"prod_R05WPWPAgXt6Kj"`

**Review by maacaro — CHANGES_REQUESTED** [review]
needs work!!!

File: frontend/src/components/PaymentGateway/PaymentGateway.tsx (hunk 1 above)
Comment (ramirezmorac2) [reply frontend/src/components/PaymentGateway/PaymentGateway.tsx:28]: Ok, I'm going to fix it now

File: frontend/src/components/PaymentGateway/PaymentGateway.tsx [hunk 2]
```diff
@@ -17,18 +18,27 @@ export const SubscriptionPlans: React.FC<{ stripePromise: Promise<Stripe | null>
 
     const [plans, setPlans] = useState<any[]>([]);
```
Comment (maacaro) [inline frontend/src/components/PaymentGateway/PaymentGateway.tsx:19]: @ramirezmorac2 we might want to remove this because it is no longer necessary

**Review by maacaro — CHANGES_REQUESTED** [review]
needs work

**Pull request merged**